docs/source/deep-learning/keras/quickstart/quickstart_keras_core.ipynb

```diff
@@ -0,0 +1,856 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "2800a1f7-80cf-47cf-83af-b117ae946a34",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Get Started with Keras 3.0 + MLflow\n",
+    "**Author:** Jesse Chan<br>\n",
```

Comment:
Our current doc does not show the author name, we can delete the author/date block.